### ISSUE TITLE
Error when I try to login

### ISSUE THREAD

**Issue by stefanopepi** [supergeorg/Grocy-SwiftUI#134]
hello all, when i try to login to my grocy instance i get this error (attached), maybe the server where i put grocy must necessarily have an ssl certificate?
[Grocy-SwiftUI_LOG.log](https://github.com/supergeorg/Grocy-SwiftUI/files/10209418/Grocy-SwiftUI_LOG.log)

**Comment by FelixSFD** [comment]
Yes. The App Transport Security prevents you from accessing HTTP-Servers.
If your server is reachable from the Internet, you can get a free SSL-Certificate from Let's Encrypt.
But it's also possible to use self-signed certificates, as long as this certificate is installed on your iOS device as well.
